src/main/scala/gitbucket/core/controller/api/ApiRepositoryCollaboratorControllerBase.scala

```diff
@@ -10,28 +10,51 @@ trait ApiRepositoryCollaboratorControllerBase extends ControllerBase {
   self: RepositoryService with AccountService with ReferrerAuthenticator with OwnerAuthenticator =>
 
   /*
-   * i. List collaborators
-   * https://developer.github.com/v3/repos/collaborators/#list-collaborators
+   * i. List repository collaborators
+   * https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#list-repository-collaborators
    */
   get("/api/v3/repos/:owner/:repository/collaborators")(referrersOnly { repository =>
     // TODO Should ApiUser take permission? getCollaboratorUserNames does not return owner group members.
     JsonFormat(
       getCollaboratorUserNames(params("owner"), params("repository")).map(u => ApiUser(getAccountByUserName(u).get))
     )
   })
+
   /*
    * ii. Check if a user is a collaborator
-   * https://developer.github.com/v3/repos/collaborators/#check-if-a-user-is-a-collaborator
+   * https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#check-if-a-user-is-a-repository-collaborator
    */
+  get("/api/v3/repos/:owner/:repository/collaborators/:userName")(referrersOnly { repository =>
+    getAccountByUserName(params("userName")) match {
+      case Some(account) =>
```

Comment:
We can use `foreach` instead of the combination of `match` and `case Some(_)` if we only need to process `Some` case. It allows omitting `None` case.

Reply:
@takezoe Thanks for your review every time! I fixed my PR! 
Finally I used `for yield` because other endpoints about collaborator use this way... Is this ok?

Reply:
Looks good!